src/components/organisms/UiList/UiList.vue

```diff
@@ -32,16 +32,47 @@
           </slot>
         </UiListItem>
       </template>
+      <!-- @slot Use this slot to replace button -->
+      <UiListItemButton v-if="hasButton">
```

Comment:
`UiListItemButton` should be an element (`UiListItem`) of list:
![image](https://github.com/infermedica/component-library/assets/37744371/56d54487-718f-4466-bcb5-5e92f4c821b2)